### PR TITLE
tests: Remove unity test from nightly test suite

### PR DESCRIPTION
**Pull request by sergiocazzolato**
As the ubuntu-clock-app is not published anymore. No snaps published by canonical anymore

### CODE REVIEW

**Comment by anonymouse64** [comment]
Can we use tests.session here instead ?
CC @zyga

**Comment by anonymouse64** [comment]
Can we use tests.session here instead ?
CC @zyga

File: tests/nightly/unity/task.yaml [hunk 1]
```diff
@@ -25,13 +25,12 @@ disabled_restore: |
 
 execute: |
     echo "Given a unity snap is installed"
-    snap install ubuntu-clock-app
+    snap install sensors-unity
```
Comment (bboozzoo) [inline tests/nightly/unity/task.yaml:28]: Should we rather use a snap that's published by us so that it does not randomly go away again?

**Comment by mvo5** [comment]
Hm, Maciej raises a good point. I would say let's disable the entire test with an explanation that there is no point in testing unity anymore and that we do not really have good test snaps anymore. This should be fine.

File: tests/nightly/unity/task.yaml (hunk 1 above)
Comment (cmatsuoka) [reply tests/nightly/unity/task.yaml:28]: Also I wonder how this will skew the metrics for that particular snap.

Comment (anonymouse64) [reply tests/nightly/unity/task.yaml:28]: all snaps in spread tests don't show up in metrics due to the SNAPPY_TESTING env var we define in spread.yaml

**Comment by sergiocazzolato** [comment]
@mvo5, I removed the test based on your comment. Thanks all for the review

**Comment by mvo5** [comment]
Thank you. We can always resurrect it if there is demand for unity snaps.

**Pull request merged**